generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.ts

```diff
@@ -157,17 +157,11 @@ export class <%= entityAngularName %>DialogComponent implements OnInit {
         this.activeModal.dismiss(result);
     }
 
-    private onSaveError(error) {
-        try {
-            error.json();
-        } catch (exception) {
-            error.message = error.text();
```

Comment:
The broken thing is.. error is of type Response, which does not even have a .message property. So the usual case - server sends JSON response - will call the alertService with "undefined" instead of a message. So this code would actually just work in the irregular case, when there is no JSON object in the server response.